### ISSUE TITLE
Minimize to tray not working

### ISSUE THREAD

**Issue by tfiers** [microsoft/terminal#12220]
### Windows Terminal version

1.11.3471.0

### Windows build number

10.0.19043.1288

### Other Software

n/a

### Steps to reproduce

1. Add `"minimizeToNotificationArea": true` to `settings.json`.
2. Restart Windows Terminal.
3. Click window's minimise button.

### Expected Behavior

Minimises to tray ("notification area")

### Actual Behavior

Minises to taskbar.

### Note

See #5727. According to the releases log this feature should be there, even in the settings UI.

**Comment by zadjii-msft** [comment]
Hmm. This might be a docs issue.

The tray icon functionality is still only available in Preview builds of the Terminal:
https://github.com/microsoft/terminal/blob/0480597abb3bacb7bc9c0d88e4ef529e4fc08e1a/src/features.xml#L67-L72

We did this to restrict the audience a bit while we sorted out remaining issues with the implementation. It's been a few months though and there haven't really been any issues with it, so maybe we should just turn that on for stable 🤔 

(tray icon did first ship in https://github.com/microsoft/terminal/releases/tag/v1.11.2421.0 but the bot was down then so none of the issue/PR have the release number posted)

(for linking #10448)
